examples/runners/runners/web_runner.py

```diff
@@ -1,49 +1,51 @@
-import json
 import logging
-import tornado.httpserver
-import tornado.ioloop
-import tornado.options
-import tornado.web
+import uvicorn
+from fastapi import FastAPI
+from pydantic import BaseModel # pylint: disable=E0611
 from surround import Runner, RunMode
 from .stages import AssemblyState
 
+app = FastAPI() # pylint: disable=C0103
+
+# Global variable to store the assembler instance (so FastAPI endpoint handlers can access it).
+assembler = None # pylint: disable=C0103
+
 logging.basicConfig(level=logging.INFO)
 
+
 class WebRunner(Runner):
 
     def load_data(self, mode, config):
         return None
 
-    def run(self, mode=RunMode.BATCH_PREDICT):
+    def run(self, mode=RunMode.PREDICT):
         self.assembler.init_assembler()
-        self.application = Application(self.assembler)
-        self.application.listen(8080)
-        logging.info("Server started at http://localhost:8080")
-        tornado.ioloop.IOLoop.instance().start()
+        global assembler # pylint: disable=C0103, W0603
```

Comment:
Would rather not use globals. I'm thinking we need to redesign the way Runners work to fix this issue.

Reply:
Yep, can do. I'll have a look at it next week :+1:

Reply:
Let's revise in a separate PR.

Reply:
Cool :+1:
We can get this in for now then and I'll address this and the lining on a separate PR

Reply:
Found a work around to replace `globals` without major restructuring